.github/workflows/ci.yml

```diff
@@ -40,6 +40,9 @@ jobs:
       - name: Compile
         run: sbt clean compile test:compile
 
+      - name: Check binary compartibility
+        run: sbt mimaReportBinaryIssues
+
```

Comment:
LGTM 👍

`sbt-dynver` can detect the previous version from a git histroy.
```
      - uses: actions/checkout@v2
        with:
          # sbt-dynver (used by sbt-ci-release) detects the previous version by tag:
          # https://github.com/dwijnand/sbt-dynver/tree/v4.1.1#previous-version-detection
          fetch-depth: 0
```
fetch-depth = 0 means that fetch all history for all tags and branches.